packages/core/package.json

```diff
@@ -17,6 +17,7 @@
         "bignumber.js": "^9.0.1",
         "react-native-keychain": "^8.0.0",
         "realm": "^10.10.1",
+        "axios": "^0.25.0",
```

Comment:
we're using `isomorphic-unfetch` in Dock Certs. Is there an advantage to one over the other? Any value in being consistent?

Reply:
Axios seems to have better tooling to mock requests, also reduces the amount of code required to handle json responses for example. I've never tried `isomorphic-unfetch` on react-native, but axios is supported without issues